scripts/generate_compiler_cpu_modifier.c

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
Missing file/copyright header, and I'm not sure this is the right name, as this file/program doesn't generate anything

Reply:
I will resolve it later.

Reply:
The commit is reverted. I found workaround for clang